src/styles/styles.js

```diff
@@ -1678,6 +1729,10 @@ function getNavigationModalCardStyle(isSmallScreenWidth) {
     };
 }
 
+function getWorkspaceSettingsDrawerType(isSmallScreenWidth) {
+    return isSmallScreenWidth ? 'back' : 'permanent';
```

Comment:
Just curious why we should want the `back` behavior instead of the `slide`? Also @shawnborton might have ideas about how this should behave at smaller screen sizes. I'm not too sure myself.

Reply:
I felt that the Sliding effect should be for the main drawer of the app. Master Drawer. and others should work as Screen stack. But yeah I do need help here to decide it.

Reply:
Reverted to main Drawer style based on Shawn's answer.